### PR TITLE
Fixed post Job Button

### PR DESCRIPTION
**Pull request by Trupti-AngDev**
# Description
Job post navigation issue was there.

# Solution
Redirected to a correct route. [/job-post]

### CODE REVIEW

File: src/app/components/navbar/navbar.component.html [hunk 1]
```diff
@@ -101,7 +101,7 @@
               Sign In</a>
           </div>
           <div class="post-job-btn mb-30">
-            <a class="primry-btn-2 lg-btn" href="#">
+            <a class="primry-btn-2 lg-btn" href="job-post">
               <svg width="15" height="13" viewBox="0 0 15 13" xmlns="http://www.w3.org/2000/svg">
```
Comment (devharp) [inline src/app/components/navbar/navbar.component.html:105]: Keep the '#' as it is. Do not change that

**Review by devharp — CHANGES_REQUESTED** [review]

File: src/app/components/navbar/navbar.component.html (hunk 1 above)
Comment (Trupti-AngDev) [reply src/app/components/navbar/navbar.component.html:105]: If we keep # as it is then routing will not be apply.

Comment (devharp) [reply src/app/components/navbar/navbar.component.html:105]: Sorry, my bad, keep it

**Review by devharp — APPROVED** [review]
Perfect

**Pull request merged**